cgrok/grok.go

```diff
@@ -196,3 +196,9 @@ func (match *Match) Captures() map[string][]string {
 
 	return captures
 }
+
+/* Returns an array of two integers, where the first is the starting index of the match, and 
+   the second is the last index of the match. */
+func (match *Match) Index() []int {
+	return []int{int(match.gm.start), int(match.gm.end)}
```

Comment:
Multiple returns are allowed (it's not quite a tuple like Python or Scala). I used this instead because it matches the standard `regexp` library's `FindIndex` call (https://golang.org/pkg/regexp/#Regexp.FindIndex). It probably makes sense to call this `FindIndex` for consistency.